assignments/assignment01.rb

```diff
@@ -38,8 +38,11 @@ def number_to_string(n, lang)
 # it accepts a string
 # and returns the same string with each word capitalized.
 def titleize(s)
-  # your implementation here
-end
+    words = s.split
+    words.each do |word|
+        word.capitalize!
```

Comment:
I encourage you to look for solutions that don't use the ! variations of methods.